pkgs/hooks/doc/debugging.md

```diff
@@ -0,0 +1,86 @@
+# Debugging Dart Build Hooks
```

Comment:
nit: technically, all that's said in here also applies to link hooks. Should we note that somewhere? Or is the omission on purpose because link hooks aren't launched yet? Even then, a short note that this also applies in the same way to the experimental link hooks may be useful.